ci/ct.yaml

```diff
@@ -0,0 +1,11 @@
+remote: origin
+helm-extra-args: --timeout 600
+check-version-increment: true
+validate-yaml: true
+
+chart-dirs:
+  - charts
+
+chart-repos:
+  - elastic=https://helm.elastic.co
```

Comment:
Why we need elastic repo here?

Reply:
Because your chart has a `requirements.yaml` that pulls in elasticsearch so it needs to know about it